firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Home/PrivateHomepageViewControllerTests.swift

```diff
@@ -0,0 +1,20 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+@testable import Client
+
+final class PrivateHomepageViewControllerTests: XCTestCase {
+    override class func setUp() {
+        super.setUp()
+        DependencyHelperMock().bootstrapDependencies()
```

Comment:
nit, can have a `tearDown()` in which you `DependencyHelperMock().reset()` as well

Reply:
updated!